### PR TITLE
nywilken/ignore prerelease plugins help text

### PR DESCRIPTION
**Pull request by nywilken**
- Add ignore-prerelease-flag to command help text
- Update CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,14 @@
             └── packer-plugin-happycloud_v0.0.1_x5.0_darwin_arm64_SHA256SUM
 ```
 
+### FEATURES:
+* cmd/build: Add `-ignore-prerelease-plugins` flag to disable the use of
+     development plugin binaries.
+     [GH-12828](https://github.com/hashicorp/packer/pull/12828)
+* cmd/validate: Add `-ignore-prerelease-plugins` flag to disable the use of
```
Comment (lbajolet-hashicorp) [inline CHANGELOG.md:26]: Should we have two entries for this? That's redundant imho
```suggestion
* cmd: Add `-ignore-prerelease-plugins` flag to disable the use of development plugin binaries for the `build` and `validate` commands
```

File: command/cli.go [hunk 2]
```diff
@@ -91,7 +91,7 @@ func (ba *BuildArgs) AddFlagSets(flags *flag.FlagSet) {
 
 	flags.BoolVar(&ba.MetaArgs.WarnOnUndeclaredVar, "warn-on-undeclared-var", false, "Show warnings for variable files containing undeclared variables.")
 
-	flags.BoolVar(&ba.ReleaseOnly, "ignore-prerelease-plugins", false, "Disable the loading of prerelease plugin binaries (x.y.z-<prerelease-name>.")
```
Comment (lbajolet-hashicorp) [inline command/cli.go:94]: Ooh good catch

**Review by lbajolet-hashicorp — APPROVED** [review]
Besides the mini nit I left, LGTM!

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,14 @@
             └── packer-plugin-happycloud_v0.0.1_x5.0_darwin_arm64_SHA256SUM
 ```
 
+### FEATURES:
+* cmd/build: Add `-ignore-prerelease-plugins` flag to disable the use of
+     development plugin binaries.
+     [GH-12828](https://github.com/hashicorp/packer/pull/12828)
```
Comment (nywilken) [inline CHANGELOG.md:25]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,6 +19,14 @@
...
+* cmd: Add `-ignore-prerelease-plugins` flag to disable the use of development plugin binaries for the `build` and `validate` commands
```
Comment (nywilken) [inline CHANGELOG.md:26]:
```suggestion
* core: Add `-ignore-prerelease-plugins` flag to disable the use of development plugin binaries for the `build` and `validate` commands
```

**Pull request merged**